doc/manual/manual.tex

```diff
@@ -330,7 +330,7 @@ \subsection{Acknowledgments}
   \end{center}
 }
 
-\section{Equations, models, coefficients}
+\section{Geodynamic Models and Numerical Methods in \aspect{}}
```

Comment:
For consistency with all other headings, don't capitalize any but the first word.

Reply:
@tjhei -- ping?